bin/good-first-issue.js

```diff
@@ -3,18 +3,20 @@
 var cli = require('commander')
 var chalk = require('chalk')
 var opn = require('open')
+const gfi = require('libgfi')
 
-var pJson = require('../package.json')
-
+var packageJSON = require('../package.json')
```

Comment:
This could be a `const`, mostly for consistency in this case!

```suggestion
const packageJSON = require('../package.json')
```